docs/sources/enterprise/access-control/permissions.md

```diff
@@ -23,62 +23,72 @@ scope
+| `settings:read`                  | `settings:*`<br>`settings:auth.saml:*`<br>`settings:auth.saml:enabled` (property level)     | Read the [Grafana configuration settings]({{< relref "../../administration/configuration/_index.md" >}})                                                   |
+| `settings:write`                 | `settings:*`<br>`settings:auth.saml:*`<br>`settings:auth.saml:enabled` (property level)     | Update any Grafana configuration settings that can be [updated at runtime]({{< relref "../../enterprise/settings-updates/_index.md" >}}).                  |
+| `server.stats:read`              | n/a                                                                                         | Read server stats                                                                                                                                          |
+| `datasources:explore`            | n/a                                                                                         | Enable access to the **Explore** tab.                                                                                                                      |
+| `datasources:read`               | n/a<br>`datasources:*`<br>`datasources:id:*`<br>`datasources:uid:*`<br>`datasources:name:*` | List data sources.                                                                                                                                         |
+| `datasources:id:read`            | `datasources:*`<br>`datasources:name:*`                                                     | Read data sources id.                                                                                                                                      |
+| `datasources:create`             | n/a                                                                                         | Create data sources.                                                                                                                                       |
+| `datasources:write`              | `datasources:*`<br>`datasources:id:*`                                                       | Update data sources.                                                                                                                                       |
+| `datasources:delete`             | `datasources:id:*`<br>`datasources:uid:*`<br>`datasources:name:*`                           | Delete data sources.                                                                                                                                       |
+| `datasources:permissions:read`   | `datasources:*`<br>`datasources:id:*`                                                       | List data sources permissions.                                                                                                                             |
```

Comment:
Minor: this one and the the ones below sound grammatically slightly odd. I'd prefer `List permissions for data sources` or `List data source permissions`. But I'm not a native speaker, so this might just be my preferences.

Reply:
```suggestion
| `datasources:permissions:read`   | `datasources:*`<br>`datasources:id:*`                                                       | List data source permissions.                                                                                                                             |
```